frontend/server/configs.ts

```diff
@@ -58,6 +58,7 @@ export function loadConfigs(argv: string[], env: ProcessEnv): UIConfigs {
     /** minio client use these to retrieve s3 objects/artifacts */
     AWS_ACCESS_KEY_ID,
     AWS_SECRET_ACCESS_KEY,
+    AWS_REGION,
```

Comment:
@zijianjoy @neerfri I think we also need to retrieve`AWS_DEFAULT_REGION`, which should take precedence over `AWS_REGION` if set.

See AWS docs for `AWS_DEFAULT_REGION` here:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html